apps/system/locales/system.en-US.properties

```diff
@@ -477,7 +477,10 @@ payment-flow=Purchase
 
 # Remote Debugger Connection Dialog
 remoteDebuggerPromptUSB=Allow USB debugging connection?
-remoteDebuggerPromptTCP=Allow remote debugging connection from {{host}}:{{port}}?
+remoteDebuggerPromptTCP.innerHTML=Allow remote debugging connection from {{host}}:{{port}}?<br/><br/>Client ID: {{cert}}
```

Comment:
Please use a more precise term like `SHA-256 certificate:`, add a line break after, and surround host and cert with `<code></code>`, like so:

    Allow remote debugging connection from <code>{{host}}:{{port}}</code>?<br/><br/>SHA-256 certificate:<br/><code>{{cert}}</code>